lunatrace/bsl/frontend/src/pages/project/builds/details/vulnerable-packages/vulnerable-package-card/PackageDetails.tsx

```diff
@@ -41,11 +41,20 @@ export const PackageDetails: React.FunctionComponent<PackageDetailsProps> = ({ p
         <Col xl={6} lg={12}>
           <h5>
             <span className="darker">Ecosystem: </span>
-            <span className="text-capitalize">{pkg.release.package.package_manager}</span>
+            <span className="text-capitalize">
+              {pkg.release.package.package_manager === 'npm' ? 'javascript' : pkg.release.package.package_manager}
+            </span>
           </h5>
           <h5>
             <span className="darker">{pluralizeIfMultiple(pkg.paths.length, 'Path') + ': '}</span>
-            <span className="lighter mx-1">{pkg.paths}</span>
+            {pkg.paths.map((path, index) => {
+              return (
+                <>
+                  {index > 0 && <br />}
```

Comment:
im not a css expert, but shouldn't this be: something, something, flex or some other css property?

Reply:
This is ok to do.  There's really no downside.  It's nice to use a span because the normal case is that there's just one path and it looks best rendered in line. Then in the 5% of times there are more paths, this separates them nicely. 

The alternative might be to apply a "d-block" class to the subsequent paths but that's more likely to mess up something else. Might work but..eh, this is fine, why complicate it?